spinnaker-monitoring/http_server.py

```diff
@@ -108,5 +108,5 @@ class HttpServer(BaseHTTPServer.HTTPServer):
 
   def __init__(self, port, handlers=None):
     BaseHTTPServer.HTTPServer.__init__(
-        self, ('localhost', port), DelegatingRequestHandler)
+        self, ('', port), DelegatingRequestHandler)
```

Comment:
I address this in another PR I sent you.
You can change the host in the spinnaker-monitoring.yml file to be empty

server:
.  host: